source/operations/install-deploy-manage/deploy-operator-kustomize.rst

```diff
@@ -0,0 +1,226 @@
+.. _minio-k8s-deploy-operator-kustomize:
+
+==============================
+Deploy Operator With Kustomize
+==============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+`Kustomize <https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization>`__ is a YAML-based templating tool that allows you to define Operator and Tenant templates using plain Kubernetes YAML reference language.
+Kustomize is included with the `kubectl <https://kubernetes.io/docs/reference/kubectl/>`__ command line tool.
+
+The `default MinIO Operator Kustomize template <https://github.com/minio/operator/blob/master/kustomization.yaml>`__ provides a starting point for customizing configurations for your local environment.
+You can modify a Kustomize Operator deployment after installation with kubectl patches, which are additional YAML configurations applied over the existing base.
```

Comment:
I think you can even include thoes patches within `kustomization.yaml`, so maybe

```suggestion
You can modify that Kustomization file *or* apply your own custom `patches <https://datatracker.ietf.org/doc/html/rfc6902>`__ to customize the Operator deployment for your Kubernetes cluster.
```

I think there might be a sphinxy way to do the RFC reference, not sure.

Reply:
Huh did not know this was a standard outside k8s 👍🏻

Reply:
FWIW I was flummoxed until I realised _why_ the K8s docs kept referencing externally for literally any guidance on how to use this.